Automattic/wp-calypso
# A4A > Agency Tier: Implement top badge section

Closes https://github.com/Automattic/automattic-for-agencies-dev/issues/1293

## Proposed Changes

This PR implements the badge section for the agency tier page.

## Why are these changes being made?

* To showcase the current agency tier for the user.

## Testing Instructions

* Open the A4A live link
* Update your agency tier to **Emerging Partner** using the MC tool > Go to Agency Tier in the UI > Verify the screen looks as shown below:

<img width="1728" alt="Screenshot 2024-10-16 at 12 47 15 PM" src="https://github.com/user-attachments/assets/974052ee-9bf5-41c9-9f65-f8320f69dc32">

* Change the agency tier to **Agency Partner** > Refresh the UI and verify the screen looks as shown below:

<img width="1728" alt="Screenshot 2024-10-16 at 12 47 27 PM" src="https://github.com/user-attachments/assets/f57ca049-571b-4110-805b-1a7b8f100909">

* Change the agency tier to **Pro Agency Partner** > Refresh the UI and verify the screen looks as shown below:

<img width="1728" alt="Screenshot 2024-10-16 at 12 47 40 PM" src="https://github.com/user-attachments/assets/4424ee9f-2d0d-4b42-9add-ef8928ad228c">

* Verify it looks good on all the screen sizes:

<img width="375" alt="Screenshot 2024-10-16 at 12 52 45 PM" src="https://github.com/user-attachments/assets/13d2e306-5255-4169-8f24-5781cd539e99">

<img width="763" alt="Screenshot 2024-10-16 at 12 54 08 PM" src="https://github.com/user-attachments/assets/545734a4-83a2-4e29-9a3d-8959be862dac">


## Pre-merge Checklist
